### DISCUSSION TITLE
CityIO

### DISCUSSION THREAD

**Discussion by raoulnay** [CityScope/CS_cityscopeJS#187] [General]
Hi @RELNO is there a date when CityIO will be live again? I couldnt find any information why it is down. Thank you!

**Comment by RELNO** [comment]
Hi @raoulnay - there's an ongoing effort to rebuild cityIO using websockets to get faster and hopefully more stable api. This is led by @amcarrero and a beta version is already live at https://cityio-beta.media.mit.edu/. Apologies for not communicating this better before. One way to still work with it is to use the cityio-flask repo as a local backend server

**Reply by raoulnay** [reply]
No problem and thank you very much!!
